FHIR-Converter/client/src/test/suite/converter/converter.test.ts

```diff
@@ -16,7 +16,7 @@ suite('Converter Test Suite', () => {
 	const activeDataPath = path.join(testPath, 'data/Hl7v2/ADT01-23.hl7');
 	const templateFolder = path.join(testPath, 'templates/Hl7v2');
 	const rootTemplate = 'ADT_A01';
-	const hl7v2Engine = new Hl7v2FhirConverterEngine(templateFolder, rootTemplate, resultFolder);
+	const hl7v2Engine = new FhirConverterEngine(templateFolder, rootTemplate, resultFolder);
```

Comment:
Are we expecting a `hl7v2Engine` or `fhirConverterEngine`?

Reply:
Here, we can think this is an Hl7v2 engine because the template folder is Hl7v2.

Reply:
Got it. Thanks!